documentation/API_documentation/population-density-data-API-Readiness-Checklist.md

```diff
@@ -1,6 +1,6 @@
 # API Readiness Checklist
 
-Checklist for population-density-data 0.1.1 in release r1.2
+Checklist for population-density-data 0.2.0-rc.1 in release r2.1
 
 | Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Comments |
```

Comment:
Template changed slightly for Spring 25, refer to https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md

E.g. add for 2 and 3 the number of the release of Commonalities and ICM the API is aligned to.

```suggestion
| Nr | API release assets  | alpha | release-candidate |  initial<br>public | stable<br> public | Status | Reference information |
```